core/server/lib/members/index.js

```diff
@@ -1,9 +1,11 @@
-const jose = require('node-jose');
 const {Router, static} = require('express');
 const body = require('body-parser');
-const jwt = require('jsonwebtoken');
 
-const cookies = require('./cookies');
+const {getData, handleError} = require('./util');
+
+const Cookies = require('./cookies');
+const Tokens = require('./tokens');
+const Users = require('./users');
```

Comment:
Unsure about calling this "users" it just gets confusing when you have "members" and a "member" or multiple of them are "members" so now you have like `/members/members` and it's not obvious what is happening :confused:

Reply:
Yeah this is a problem, "users" might become confusing as well after being exposed outside 😕 Don't have any better naming ideas tbh, maybe "memberUsers"(?)  or "memberships"(?) 😬Can leave this for longer term but this will def keep coming back

Reply:
Yeah possibly :confused: Maybe we want to change "members" to "memberships" and "users" to "members" :man_shrugging: